### PR TITLE
Setup cob4 paul stuttgart

### PR DESCRIPTION
**Pull request by NadiaHG**
- paul configuration for cob4-7 
 - remove cob4-10 robot,not needed (this was cob4-7 with arms)

### CODE REVIEW

**Comment by NadiaHG** [comment]
This PR is connected to https://github.com/ipa320/cob_calibration_data/pull/128 and https://github.com/ipa320/cob_supported_robots/pull/3

**Comment by fmessmer** [comment]
requires https://github.com/ipa320/cob_supported_robots/pull/3
requires https://github.com/ipa320/cob_calibration_data/pull/128

one more reason, why we need to proceed with #622 soon :wink:

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py [hunk 1]
```diff
@@ -89,6 +89,57 @@ def front_to_home_cb(req):
...
+        handle_base = sss.move_base_rel("base",[0,0,-1.2],False)
+        sss.sleep(2)
+        handle_arm_left = sss.move("arm_left","side", False)
+        handle_arm_right = sss.move("arm_right","side", False)
+        sss.set_mimic("mimic",["blinking_right",0,1])
+        handle_base.wait()
+    else:
+        return TriggerResponse(False, "Could not move arms.")
+
     return TriggerResponse(True, "")
```
Comment (fmessmer) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py]: cob4-7 has no arms (for the time being)..remove `pick_cb` (is buggy anyway)

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py [hunk 2]
```diff
@@ -204,6 +255,7 @@ def trigger_srvs():
     rospy.init_node('trigger_srvs')
     s = rospy.Service('/behavior/torso_front', Trigger, torso_front_cb)
     s = rospy.Service('/behavior/torso_front_home', Trigger, front_to_home_cb)
+    s = rospy.Service('/behavior/pick', Trigger, pick_cb)
```
Comment (fmessmer) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py]: remove pick

File: cob_hardware_config/cob4-7/config/diagnostics_analyzers.yaml [hunk 3]
```diff
@@ -105,22 +105,14 @@ analyzers:
...
         path: Sensorring
         contains: 'sensorring/'
-      head:
-        type: diagnostic_aggregator/GenericAnalyzer
-        path: Head
-        contains: 'head/'
+#      head:
+#        type: diagnostic_aggregator/GenericAnalyzer
+#        path: Head
+#        contains: 'head/'
```
Comment (fmessmer) [inline cob_hardware_config/cob4-7/config/diagnostics_analyzers.yaml]: remove bms, sensorring and head from this file

File: cob_hardware_config/cob4-7/config/joint_state_publisher.yaml [hunk 4]
```diff
@@ -1,2 +1,2 @@
 rate: 100 #Hz
-source_list: [/base/joint_states, /torso/joint_states, /arm_left/joint_states, /arm_right/joint_states, /sensorring/joint_states, /head/joint_states]
+source_list: [/base/joint_states, /torso/joint_states, /sensorring/joint_states, /head/joint_states]
```
Comment (fmessmer) [inline cob_hardware_config/cob4-7/config/joint_state_publisher.yaml]: remove sensorring and head

File: cob_hardware_config/cob4-7/config/teleop.yaml [hunk 5]
```diff
@@ -56,48 +44,24 @@ right_left_button: 4
...
+#    sss_default_target: "home",
+#    velocity_topic_name: "/torso/joint_group_velocity_controller/command",
+#    joint_velocity: [0.05, 0.05],
+#    twist_topic_name: '/torso/twist_controller/command_twist',
+#    twist_max_velocity: [0.3, 0.2, 0.3, 0.3, 0.2, 0.3]
+#  },
+#  sensorring: {
+#    sss_default_target: "front",
+#    velocity_topic_name: "/sensorring/joint_group_velocity_controller/command",
+#    joint_velocity: [0.05]
```
Comment (fmessmer) [inline cob_hardware_config/cob4-7/config/teleop.yaml]: remove sensorring?

File: cob_hardware_config/cob4-7/config/teleop.yaml [hunk 6]
```diff
@@ -32,22 +32,10 @@ run_button: 7
 #mode2: Trajectory controller
 torso_sss_default_target_button: 1
 sensorring_sss_default_target_button: 3
```
Comment (fmessmer) [inline cob_hardware_config/cob4-7/config/teleop.yaml]: remove sensorring?

File: cob_hardware_config/cob4-7/config/teleop.yaml [hunk 7]
```diff
@@ -32,22 +32,10 @@ run_button: 7
...
-arm_right_sss_default_target_button: 2
-arm_left_sss_default_target_button: 0
-head_sss_default_target_button: 3
 
 #mode3: Velocity group controller
 torso_joint1_button: 0
 sensorring_joint1_button: 1
```
Comment (fmessmer) [inline cob_hardware_config/cob4-7/config/teleop.yaml]: remove sensorring?

**Review by fmessmer — CHANGES_REQUESTED** [review]
missed a few spots...

File: cob_hardware_config/cob4-7/config/teleop.yaml [hunk 8]
```diff
@@ -56,48 +42,19 @@ right_left_button: 4
...
-    twist_max_velocity: [0.3, 0.2, 0.3, 0.3, 0.2, 0.3]
-  }
+#  },
+#  torso: {
+#    sss_default_target: "home",
+#    velocity_topic_name: "/torso/joint_group_velocity_controller/command",
+#    joint_velocity: [0.05, 0.05],
+#    twist_topic_name: '/torso/twist_controller/command_twist',
+#    twist_max_velocity: [0.3, 0.2, 0.3, 0.3, 0.2, 0.3]
+#  }
```
Comment (fmessmer) [inline cob_hardware_config/cob4-7/config/teleop.yaml]: now there is one closing `}` missing

**Review by fmessmer — CHANGES_REQUESTED** [review]

**Review by fmessmer — APPROVED** [review]

**Comment by fmessmer** [comment]
rebased, squashed, sorted and merged via #635

**Pull request closed without merging**